lib/board.rb

```diff
@@ -0,0 +1,57 @@
+class Board 
+  attr_accessor :cells, :token
+  
+ def reset!
+   @cells=Array.new(9, " ")
+  
+  end 
+  
+  def initialize
+    reset!
+    
+  end
+  
+  def display
+  puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+  puts "-----------"
+  puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+  puts "-----------"
+  puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+    
+  end 
+  
+  def position(input) 
+    cells[input.to_i-1]
+  end 
+  
+  def update(input,player)
+   cells[input.to_i-1] = player.token
+  end 
+  
+ def full?
+    if cells.include?(" ")
+      return false
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression.